src/main/java/blob/tasks/Event.java

```diff
@@ -0,0 +1,43 @@
+package blob.tasks;
+
+import blob.parser.DateTimeParser;
+import blob.exception.InvalidDateFormatException;
+
+import java.time.format.DateTimeParseException;
+
+/**
+ * The Event class encapsulates a type of task to be done.
+ * A Deadline task is a task that starts at a specific time and ends at a specific time.
+ */
+public class Event extends Task {
+    private String at;
+
+    public Event(String description, String at) throws InvalidDateFormatException {
+        super(description, TaskType.EVENT);
+        try {
+            this.at = new DateTimeParser().parse(at);
```

Comment:
It is great that this the "at" date of events is also parsed. Would the bot be expecting the same formats for both events and deadlines? If so, this would work well. Otherwise, one suggestion may be to create a separate "at" parser for events.